README.md

```
@@ -74,6 +75,13 @@ This platform is designed to empower users in managing waste efficiently and con
 
 ---
 
+## Contributors
+A huge thanks to all our contributors 
+
+<a href="https://github.com/GarimaSingh0109/WasteManagment/graphs/contributors">
+  <img src="https://contrib.rocks/image?repo=GarimaSingh0109/WasteManagment" />
```

Markdownlint (0.35.0)

Images should have alternate text (alt text)

(MD045, no-alt-text)